.env.baklavastaging

```diff
@@ -0,0 +1,107 @@
+# Don't use "//" for comments in this file.
+# This file is meant to be executed as a bash script for testing.
+ENV_TYPE="staging"
+
+GETH_VERBOSITY=2
+
+VM_BASED=true
+
+KUBERNETES_CLUSTER_NAME="baklavastaging"
+KUBERNETES_CLUSTER_ZONE="us-west1-a"
+CLUSTER_DOMAIN_NAME="celo-testnet"
+
+TESTNET_PROJECT_NAME="celo-testnet"
+
+BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="ad86714d629c01272e0651dec1fb6a968c3cec71"
+BLOCKSCOUT_WEB_REPLICAS=3
+# Increment this value everytime you redeploy blockscout. Or else the deployment will fail due to the
+# existing database.
+BLOCKSCOUT_DB_SUFFIX="0"
+BLOCKSCOUT_SUBNETWORK_NAME="Baklava Staging"
+
+ETHSTATS_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/ethstats"
+ETHSTATS_DOCKER_IMAGE_TAG="fb9bcc639eeb0d3a70dadaf2aa3c49d45c7d9f5c"
+
+GETH_NODE_DOCKER_IMAGE_REPOSITORY="us.gcr.io/celo-testnet/geth"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_NODE_DOCKER_IMAGE_TAG="ed1312266c350a53316ed82c47c52327f227b727"
+
+GETH_BOOTNODE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/geth-all"
+# When upgrading change this to latest commit hash from the master of the geth repo
+# `geth $ git show | head -n 1`
+GETH_BOOTNODE_DOCKER_IMAGE_TAG="ed1312266c350a53316ed82c47c52327f227b727"
+
+CELOTOOL_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/celo-monorepo"
+CELOTOOL_DOCKER_IMAGE_TAG="celotool-bffefdb9e94a64b6e70a90aab60dd85ca39d7862"
+
+CELOCLI_STANDALONE_IMAGE_REPOSITORY="gcr.io/celo-testnet/celocli-standalone"
+CELOCLI_STANDALONE_IMAGE_TAG="0.0.30-beta2"
+
+# Schedule for an oracle deployed via celotool, expressed in crontab syntax
+# This schedule is "every 5th minute"
+ORACLE_CRON_SCHEDULE="*/5 * * * *"
+
+ORACLE_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/oracle"
+ORACLE_DOCKER_IMAGE_TAG="baklavastaging"
```

Comment:
The origin of this file is @tkporter's branch for baklavastaging deployment. It's possible that some vars here may have changed since I copied this over. The only changes I made that are required for the oracle deployment are these highlighted variables.